cc/UWU/2022-current.md

```diff
@@ -0,0 +1,32 @@
+# UWU - Uva Wellassa University ![Final_Logo](https://user-images.githubusercontent.com/68012743/202668062-e0190d81-9b80-4c03-8692-4241d2a1f41a.png)
+
```

Comment:
```suggestion
<img src="https://user-images.githubusercontent.com/68012743/202668062-e0190d81-9b80-4c03-8692-4241d2a1f41a.png" width="20%" />
```

Adjust width accordingly

Reply:
@ranidu-harshana why was this marked resolved?

Reply:
Sorry for the mistake aiya